hooks/react-client/useDetectOS.ts

```diff
@@ -31,7 +31,12 @@ const useDetectOS = () => {
           bitness:
             bitness === '64' ||
             userAgent?.includes('WOW64') ||
-            userAgent?.includes('Win64')
+            userAgent?.includes('Win64') ||
```

Comment:
Could you make a simple regex with something like `/WOW64|Win64|x86_64|x86-64|x64;|AMD64/.test(userAgenmt)`?

Maybe worth also assigning this Regex into a dedicated const